scripts/docker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+# Hack that helps with the cache
+docker pull $(grep -ioP '(?<=^from)\s+\S+' docker/spark/Dockerfile)
+docker pull metorikku/spark:latest
+docker pull metorikku/metorikku:latest
+docker pull $(grep -ioP '(?<=^from)\s+\S+' docker/hive/Dockerfile)
+docker pull metorikku/hive:latest
+docker pull $(grep -ioP '(?<=^from)\s+\S+' docker/hive1/Dockerfile)
+docker pull metorikku/hive:1
```

Comment:
1?
is the on purpose?

Reply:
Well yeah, we support both hive1 and hive2, we tag hive1 as hive:1